packages/palette/src/elements/Link/Link.tsx

```diff
@@ -29,7 +29,8 @@ const backwardsCompatCompute = (state: string, props: LinkProps) => {
 /**
  * Basic <a> tag styled with additional LinkProps
  *
- * @deprecated If working in Force, please use `RouterLink`.
+ * @deprecated don't use this component anymore
```

Comment:
Maybe something a little more forceful since this issue comes up soooo often? 

```suggestion
 * @deprecated Do not use this component! 
```